src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -151,6 +151,7 @@ class CompilerContainer {
       <select onchange="${this.onchangeEvmVersion.bind(this)}" class="custom-select" id="evmVersionSelector">
         <option value="default">compiler default</option>
         <option>petersburg</option>
+        <option>istanbul</option>
```

Comment:
IMHO it should be above `petersburg` because its never.

Reply:
good point